core/components/atoms/dropdown/__stories__/variants/CustomOption.story.jsx

```diff
@@ -32,21 +23,21 @@ export const customOption = () => {
     const { label } = props.optionData;
 
     return (
-      <div style={optionStyle}>
+      <div className="px-5 py-4 d-flex cursor-pointer justify-content-between">
         <Checkbox checked={props.selected} onChange={props.onChange} label={label} />
         {props.selected && <Icon name="done" />}
       </div>
     );
   };
 
   return (
-    <div style={{ display: 'flex', minHeight: '280px' }}>
-      <div style={{ marginRight: '20%', width: '200px' }}>
+    <div className="d-flex pb-14">
+      <div className="Dropdown-spacing w-25">
```

Comment:
Please use className `Row Col` or `width` classes instead of using margin-right class